evalutils/templates/evaluation/{{ cookiecutter.package_name }}/test.sh

```diff
@@ -1,21 +1,23 @@
 #!/usr/bin/env bash
 
-SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
```

Comment:
This line used to `cd`, it no longer does. The new L7 probably needs $SCRIPTPATH prepended.